src/content/configuration/stats.mdx

```diff
@@ -539,19 +539,23 @@ module.exports = {
 
 ### stats.context
 
-`string = '../src/'`
+`string`
 
-Sets the context directory for shortening the request information.
+The stats base directory, an **absolute path** for shortening the request information.
 
 ```javascript
+const path = require('path');
+
 module.exports = {
   //...
   stats: {
-    context: '../src/components/',
+    context: path.resolve(__dirname, 'src/components'),
   },
 };
 ```
 
+By default, the value of [`context`](/configuration/entry-context/#context) or the Node.js current working directory is used.
```

Comment:
Can you provide the source for this information?

Reply:
@snitin315 the [stats default options](https://github.com/webpack/webpack/blob/937957fc545fae96a15ce0ea3fb9b7910de41f7e/lib/stats/DefaultStatsPresetPlugin.js#L135) and [`context` info](https://github.com/webpack/webpack.js.org/blob/6dcf4ceb6f731ad62433706ff788f109ca2cd16a/src/content/configuration/stats.mdx#statscontext)